src/Search.js

```diff
@@ -10,20 +10,26 @@ class Search extends Component {
   }
     handleChange = (event) => {
       this.setState({
-        countryName: event.target.value
+        countryName: event.target.value.toLowerCase()
       });
     }
 
     takeCountry = (event, country) => {
       event.preventDefault();  
-      document.querySelector('.input').value = '';
+      this.props.updateCountryInput(this.state.countryName)
+      this.props.findCountry(this.state.countryName);
+      this.setState({
+        countryName: ''
+      })
```

Comment:
Passing up our data that we received in our search components state into the 'findCountry()' method that was passed down as props. Now our method is being invoked to find the country the user has searched for.